components/Landing/Creators.tsx

```diff
@@ -73,26 +73,31 @@ export const Creators: React.FC = () => {
             "empty1 admins empty2 empty2"
           `,
         }}
+        fontSize={{ base: 'md', md: '2xl' }}
       >
         {!isSmallScreen && (
-          <Box h="338px" gridArea="creators" pr={12}>
+          <Box h="338px" gridArea="creators" mb={40} mr={10}>
```

Comment:
here as you can see the "reviwers" block needs to be linked to the other roles
![Screenshot from 2022-10-24 21:20:41](https://user-images.githubusercontent.com/10012209/197570050-979e6d0a-fd29-458f-98a8-0b0ed66dce58.png)